src/lib/x509/certstor_system_linux/info.txt

```diff
@@ -0,0 +1,16 @@
+<defines>
+CERTSTOR_LINUX -> 20190402
+</defines>
+
+<os_features>
+linux_certstore
```

Comment:
This should be

```
<os>
linux
</os>

<requires>
certstore_flatfile
</requires>
```

Reply:
This confuses `configure.py`:

```
LexerError: Unknown group "os" at ../src/lib/x509/certstor_system_linux/info.txt:6
```

Reply:
Sorry my bad, actually `<os>` that was removed a while back.